src/multicloud-connector/azext_multicloud_connector/aaz/latest/arc_multicloud/solution_configuration/_create.py

```diff
@@ -18,7 +18,10 @@ class Create(AAZCommand):
     """Create a SolutionConfiguration
 
     :example: SolutionConfigurations_CreateOrUpdate
-        az arc-multicloud solution-configuration create --connector-id ymuj --name keebwujt --solution-type nmtqllkyohwtsthxaimsye --solution-settings "{}"
+        az arc-multicloud solution-configuration create --connector-id /subscriptions/{}/resourceGroups/{}/providers/Microsoft.HybridConnectivity/publicCloudConnectors/{} --name mySolutionConfig --solution-type "Microsoft.AssetManagement" --solution-settings periodicSync="true" cloudProviderServiceTypes="ec2,s3" awsGlobalReadOnly="true" cloudProviderRegions="us-east-1,us-east-2" periodicSyncTime="1"
```

Comment:
Try to capture all the available options 

![image](https://github.com/user-attachments/assets/22c389f1-deeb-44a8-adc4-5f895e3064ce)

Reply:
Also create a work item to link the public documentation that lists all the available options with default values. Please work with Meagan on this.